src/containers/app/AppFooter.js

```diff
@@ -29,18 +30,31 @@ export function AppFooter(props) {
                 </Segment>
                 <Segment basic as="p" className="description">
                     {description}
-                </Segment>
-                <Segment
-                    basic
-                    className="join-discord"
-                    size="tiny"
-                    as="a"
-                    href="https://discord.gg/PwDmsnu"
-                    target="_blank"
-                >
-                    <Image src={discordLogo} />
-                    Talk to us on Discord
-                </Segment>
+                </Segment >
+                <Segment basic className="chat-container">
+                    <Segment
+                        basic
+                        className="join-discord"
+                        size="tiny"
+                        as="a"
+                        href="https://discord.gg/PwDmsnu"
+                        target="_blank"
+                    >
+                        <Image src={discordLogo} />
+                        Talk to us on Discord
+                    </Segment>
+                    <Segment
+                        basic
+                        className="join-telegram"
+                        size="tiny"
+                        as="a"
+                        href="https://t.me/augmint"
+                        target="_blank"
+                    >
+                        <Image src={telegramLogo} />
+                        Talk to us on Telegram
```

Comment:
can we say `Talk with us on Telegram`?

Reply:
@szerintedmi told me, we talk about it later